content/docs/start/index.md

```diff
@@ -67,6 +67,11 @@ by one in the next few pages:
   data artifacts are built, and provide an efficient way to reproduce them.
   Think "Makefiles for data and ML projects" done right.
 
-- [**Experiments**](/doc/start/experiments) attach parameters, metrics, plots.
-  You can capture and navigate experiments without leaving Git. Think "Git for
-  machine learning".
+- [**Metrics, parameters, and plots**](/doc/start/metrics-parameters-plots)
+  explains how to attach metrics, parameters, and plots. You can capture,
```

Comment:
Metrics, parameters, and plots explains how to attach metrics, parameters, and plots — not v informative 🙃

Maybe something like `[Parameters, metrics, and plots](...) can be attached to pipelines. These let you capture...`